riscv-runtime/src/io.rs

```diff
@@ -51,6 +141,16 @@ pub fn write_slice(fd: u32, data: &[u8]) {
 use serde::de::DeserializeOwned;
 use serde::Serialize;
 
+pub fn read_stdin<T: DeserializeOwned>() -> T {
+    let mut reader = get_prover_data_reader();
+    if let Some(ref mut reader) = *reader {
+        let slice = reader.next().unwrap();
+        bincode::deserialize(slice).unwrap()
+    } else {
+        panic!("powdr ProverDataReader not available");
```

Comment:
what happens if thsi is called outside of continuations?

Reply:
I think it will panic because the data hasn't been given, similar to the channel case